compiler/grainlsp/rpc.re

```diff
@@ -0,0 +1,411 @@
+let jsonrpc = "2.0";
+
+type protocol_msg =
+  | Message(int, string, Yojson.Safe.t)
+  | Error(string)
+  | Notification(string, Yojson.Safe.t);
+
+[@deriving yojson]
+type completion_values = {
+  resolveProvider: bool,
+  triggerCharacters: list(string),
+};
+
+// [@deriving yojson]
+// type signature_helpers = {
+//   triggerCharacters: list(string),
+//   retriggerCharacters: list(string),
+// };
```

Comment:
What's this?

Reply:
Moved to a new PR that implements signature helpers